File: arcore/src/main/java/com/hiscene/arcore/AppApplication.java
package com.hiscene.arcore;

import android.app.Application;

import com.google.ar.core.Session;

/**
 * Created by hujun on 2018/12/17.
 */

public class AppApplication extends Application {
    private Session arSession;
    @Override
    public void onCreate() {
        super.onCreate();
    }

    public Session getArSession(){
        return arSession;
    }

    public void setArSession(Session session){
        arSession = session;
    }
}

File: Demo2/architecturecomponent/src/main/java/com/hiscene/architecturecomponent/AppApplication.kt
package com.hiscene.architecturecomponent

import android.app.Application

/**
 * Created by junhu on 2019-11-05
 */
class AppApplication:Application() {
    companion object{
        lateinit var instance:Application
    }
    override fun onCreate() {
        super.onCreate()
        instance = this
    }
}